packages/react-components/src/components/utils/SendBoxUtils.ts

```diff
@@ -14,21 +14,21 @@ const EMPTY_MESSAGE_REGEX = /^\s*$/;
 /**
  * @private
  */
-export const hasIncompleteFileUploads = (activeFileUploads: ActiveFileUpload[] | undefined): boolean => {
+export const hasIncompleteAttachmentUploads = (activeAttachmentUploads: AttachmentMetadata[] | undefined): boolean => {
   return !!(
-    activeFileUploads?.length &&
-    !activeFileUploads
-      .filter((attachmentUpload) => !attachmentUpload.error)
-      .every((attachmentUpload) => attachmentUpload.uploadComplete)
+    activeAttachmentUploads?.length &&
+    !activeAttachmentUploads
+      .filter((attachmentUpload) => !attachmentUpload.uploadError)
+      .every((attachmentUpload) => attachmentUpload.progress === 1 || !attachmentUpload.progress)
```

Comment:
I think if  `attachmentUpload.progress` equals `0`, `!attachmentUpload.progress` will return true but it's not what expected here